bundle/manifests/kuadrant-operator.clusterserviceversion.yaml

```diff
@@ -449,21 +407,6 @@ spec:
           - patch
           - update
           - watch
-        - apiGroups:
-          - rbac.authorization.k8s.io
```

Comment:
not needed

```
❯ k auth can-i create roles --as=system:serviceaccount:kuadrant-system:kuadrant-operator-controller-manager -n kuadrant-system
no
```
But the Kuadrant, RL and AuthPolicy controllers work as expected (following [Authenticated Rate Limit For API Owners](https://github.com/Kuadrant/kuadrant-operator/blob/e22eff65ef2d0ad5d42cdd50f5299f9ce0f22272/doc/user-guides/authenticated-rl-for-api-owners.md#authenticated-rate-limit-for-api-owners)